apps/charterafrica/contrib/dokku/Dockerfile

```
@@ -1 +1 @@
-FROM codeforafrica/charterafrica-ui:0.0.15
+FROM codeforafrica/charterafrica-ui:0.0.16
```

```suggestion
FROM codeforafrica/charterafrica-ui:0.0.21
```

I think we've already create images up to `0.0.19`/`0.0.20`. Lets not go back.

---

Moved to0.0.22, we have 0.0.21